### ISSUE TITLE
Proposal to Improve Express Website/Examples/Docs

### ISSUE THREAD

**Issue by tinafangkunding** [expressjs/expressjs.com#1176]
## Problem & Background

Hi there, we are Express.js users, and also serverless advocates. We found there are some issues about "lack of introduction to examples/cloud environment deploy issues", as the following ones:

https://github.com/expressjs/express/issues/4343
https://github.com/expressjs/express/issues/3468
https://github.com/expressjs/express/issues/4278 

Therefore, in this issue, we’d like to collaborate with the express community and make a better website/content for users. First of all, we’d like to focus on adding some guidance to people who want to deploy express.js on cloud environments, especially on serverless architecture. Besides, we’d like to discuss how to help users create an express.js app quickly and easily, especially for Chinese users (express.js is very popular in China). 
 
This proposal based on the following principles:
- **Open-Source:** all the content and tools used (e.g. the Serverless Framework) is open-sourced.
- **Multi-Cloud Environment Support:** We’ll make sure the content is neutral and the examples can be deployed to Multi-Cloud Environment. 

## Proposed Solutions

After a discussion with @dougwilson, I slightly changed the solution and focus on the express website.  Looking forward to your suggestions and feel free to add more thoughts here! cc @Gbahdeyboh @OmegaCrack

1. **Maintain/Update the content in Express.js website.** https://www.expressjs.com
There are many resources on the express website, some tutorial such as `quick start` is pretty neat, while some other modules seem not updated/maintained for a while, such as `Advance topics` and `Resources`. Maybe we can look into them and remove some outdated content.

2. **Update and Maintain Examples/Templates in Repo:** 
As a user who wants to learn express.js quickly and conveniently, I think some pre-set templates will help a lot during the learning. <br>
For example, if a user wants to build a blog based on express, he can easily download the templates and develop/deploy in a local or cloud environment. Then the blog is out of the box, and the user can keep developing the blog and add customized content. It's pretty streamlined experience for new learners. <br>
Thus, we’d like to provide some typical examples, and these examples can be easily deployed to Multi-Cloud environments via the Serverless Framework, one of the most popular open-source development tools. Here are some examples we’d like to deliver:

    - Express.js RESTful API 
    - Websocket Demo based on Express.js
    - OCR application based on Express.js
    - Translation Application based on Express.js
    - Express.js + PostgreSQL example
    - Express.js + MySQL example
    - Express.js + MongoDB example
    - ...

3. **Chinese Express.js website improvements**: 
Currently, the website https://www.expressjs.com.cn/ is the top result in China’s Search Engine, which means it’s an important entrance to know express.js for Chinese users. While the real official site http://expressjs.com/zh-cn/ is not shown as a result in the Search Engine
![image](https://user-images.githubusercontent.com/26135229/87288498-03bb5e80-c52e-11ea-8f99-ab85c3717d40.png)
With the improvements/templates update, we'd like to update the content in the official site http://expressjs.com/zh-cn/ and make the machine translation more accurate. Besides, we should improve the SEO result and let Chinese users know this site's content is the official, well-maintained one.

## Express in China
According to a [node.js survey](https://nodersurvey.github.io/reporters/#%E6%A1%86%E6%9E%B6) in China 2020, Express.js is the most popular node.js framework (61.37%) in China, and many of the users deploy their express.js applications on the cloud environment, it will be great if we can support more content in the official website/repo. 

![image](https://user-images.githubusercontent.com/26135229/87290174-3e25fb00-c530-11ea-8b1c-3e57279d59db.png)

Looking forward to your suggestions, and please let us know your thoughts from the community side, thanks in advance!

## Ref
The original issue link: https://github.com/expressjs/express/issues/4345

**Comment by jason-alouda** [comment]
Hi, I'm currently working with Tina on this proposal. Where should we upload the examples on the Github repo https://github.com/expressjs/expressjs.com? The first example we want to upload is a simple Express.js RESTful API that can be easily deployed to Multi-Cloud environments via the Serverless Framework, one of the most popular open-source development tools. Thanks a lot!

**Comment by crandmck** [comment]
The only place we have for example code is https://github.com/expressjs/express/tree/master/examples, but if Doug directed you to the website, here's what I'd suggest: Create the example in your own standalone repo and then open a doc PR that explains the code, with snippets, links to the repo, etc.  This page currently links to the examples in the code repo, so it would be the logical place to add further standalone examples.  Once your doc PR lands, at some point you could open a separate PR to add your examples to express/examples.
